README.md

```diff
@@ -1,5 +1,6 @@
 # [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) LORIS Neuroimaging Platform 
 
+<!--alex ignore easy-->
```

Comment:
The word 'easy' is not helpful to include in code instructions because different tasks are more difficult depending on your background.

However, it's a nice "sales" line in the context of this paragraph